src/services/FulfillmentService.js

```diff
@@ -29,12 +30,17 @@ class FulfillmentService {
     try {
       const {
         params: { slug },
-        body: { type, value },
+        body: { location },
       } = req;
-
-      // TODO: get items by slug,type and value
-
-      return organizations;
+      const county = location ? location.toUpperCase() : '';
+      const url = `${process.env.PROJECT_URL}/rest/v1/fulfillments/services/${county}`;
```

Comment:
There will be many different types of services that we will need to search for based on the slug that was passed in. Let's talk about this tomorrow morning.